scenariolib/event_pageview.go

```diff
@@ -83,6 +93,13 @@ func (view *ViewEvent) send(v *Visit) error {
 	return v.SendViewEvent(event)
 }
 
+func computePageViewField(view *ViewEvent, v *Visit) string {
```

Comment:
With the Go JSON parsing, since `pageViewField` is obligatory, the event will fail to parse with an error if you do not specify a field.

You can still specify manually `"pageViewField": ""`. So if you want to make sure there is a value, you could just do:

```
if view.PageViewField == "" {
  view.PageViewField = v.Config.DefaultPageViewField
  Warning.Printf("PageViewField was empty, using default value, %s", v.Config.DefaultPageViewField)
}
```

Reply:
> With the Go JSON parsing, since pageViewField is obligatory, the event will fail to parse with an error if you do not specify a field.

🤔 My scenario does not have this field, so it should be failing, but it is not... Are you certain of this?

Reply:
Hmmm yes, I believe I have done local tests with different events...

Maybe it just applies the nil value?

Reply:
Yea... so side effect of this is that it means that if you don't specify a `docNo` it will click on the result at position 0.